### DISCUSSION TITLE
v4.1.0

### DISCUSSION THREAD

**Discussion by coolapso** [streamdeck-linux-gui/streamdeck-linux-gui#148] [Announcements]
Hey there,

It's been a while! I'm happy to let you know that v4.1.0 is out now. Check out the release notes here: [v4.1.0 Release Notes](https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/releases/tag/v4.1.0).

This update includes a few things: some bug fixes and updates to dependencies. Most importantly, we've started adding support for Stream Deck +. As of now, the touchscreen and rotary buttons aren't working yet, but the standard buttons are up and running.

Cheers,

**Comment by oparada1988** [comment]
Thank you for all of your hard work!

**Comment by Ben-PP** [comment]
Thank you for this! I can finally use my old streamdeck after the switch to linux! Looking forward to the addition of streamdeck+
